augur/util_support/node_data_file.py

```diff
@@ -9,8 +9,9 @@
 
 
 class NodeDataFile:
-    def __init__(self, fname):
+    def __init__(self, fname, skip_validation=False):
```

Comment:
I agree it's annoying to pass around this variable through multiple classes. This isn't ideal OO design and an OO implementation is even what we need here. Your implementation here seems fine, in the current context. We should leave to another PR to reimplement the node data reader as a function again. 😄

Reply:
Yes!